src/firebase/AuthService.ts

```diff
@@ -15,13 +15,29 @@ class AuthService {
         });
     }
 
+    private async createUserProfileIfRequired() {
+        const user = firebase.auth().currentUser;
+        if (user) {
+            const id = user.uid;
+            const db = firebase.firestore();
+            const docRef = db.collection("users").doc(id);
+            const doc = await docRef.get();
+            if (!doc.exists) {
+                db.collection("users").doc(id).set({
+                    isAdmin: false,
+                    isSnappable: true
```

Comment:
This should be false. We don't want everyone who logs in to be automatically snappable.